intro-to-tidyverse.rmd

```diff
@@ -1007,17 +1033,59 @@ tidy_df %>%
 ```{r, eval = FALSE, highlight.output = 2}
 tidy_df %>% 
   unite("gender_year", c("gender", "year"), sep = "_") %>% 
-  spread(gender_year, income) #<<
+  pivot_wider(names_from = gender_year, values_from = income) #<<
 ```
 
 --
 
 ```{r, echo = FALSE, highlight.output = 2}
 tidy_df %>% 
   unite("gender_year", c("gender", "year"), sep = "_") %>% 
-  spread(gender_year, income) #<<
+  pivot_wider(names_from = gender_year, values_from = income)#<<
+```
+
+---
+
+class: center, middle, inverse
+
+# SIDEBAR!
```

Comment:
Including janitor as a way to talk about the importance of consistent styling and what I would consider tidy col names